### PR TITLE
Document `license_model` as supported argument

### PR DESCRIPTION
**Pull request by ijin**
Required for Oracle SE1

### CODE REVIEW

File: website/source/docs/providers/aws/r/db_instance.html.markdown [hunk 1]
```diff
@@ -81,6 +81,7 @@ database, and to use this value as the source database. This correlates to the
 [Working with PostgreSQL and MySQL Read Replicas](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ReadRepl.html) for
  more information on using Replication.
 * `snapshot_identifier` - (Optional) Specifies whether or not to create this database from a snapshot. This correlates to the snapshot ID you'd find in the RDS console, e.g: rds:production-2015-06-26-06-05.
+* `license_model` - (Optinal, but required for some DB engines, i.e. Oracle SE1) License model information for this DB instance.
```
Comment (stack72) [inline website/source/docs/providers/aws/r/db_instance.html.markdown]: There is a spelling mistake in Optional

**Comment by ijin** [comment]
@stack72 thanks, fixed and rebased!

**Comment by stack72** [comment]
@ijin thanks :)

This is as per the [documention](http://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance.html)

**Comment by jen20** [comment]
:+1: thanks the PR @ijin!

**Pull request merged**